week_3_alt_assignment/index.js

```diff
@@ -0,0 +1,307 @@
+/**
+ ******************************************************************************
+ *********************** Assignment Instructions! *****************************
+ ******************************************************************************
+ *
+ * First, start up the website and check it out. This is an interactive
+ * tutorial, so interact with it before you go any further! You can run this
+ * server just like the assignments you have been working on so far:
+ *
+ * ```
+ * npm install
+ * npm run start
+ * ```
+ *
+ * Go through the website, then come back; have fun!
```

Comment:
> I found this alternative assignment useful and fun because it's interactive and the students can use their creativity to create the content of their HTML page. I was a bit confused at first because I missed opening the link written on [line 62 ](http://localhost:3000/whatisaserver/html)on index.js But overall it's a good assignment even I can take advantage of.

@larasodjati  Thanks for the feedback! I added some ascii art around the link to make it way harder to miss!